### ISSUE TITLE
Weird blue world on the 0.48 RC

### ISSUE THREAD

**Issue by Cloudy100** [zesterer/openmw-shaders#25]
![weirdissue](https://user-images.githubusercontent.com/51799285/189044632-b4b9c865-ccde-4cb3-9563-ed237fb89d56.PNG)

the release canditate just came out, and I tested it and got this.

**Comment by zesterer** [comment]
Thanks for reporting, I'll try to find the time to look into this over the next few days.

**Comment by XC3N** [comment]
I couldn't reproduce this with 0.48 RC2, however I have had similar looking issues when my openMW advanced visual settings were not set properly.

Can you make sure you are using Shaders for lighting and that pretty much everything in advanced/visuals is checked?

Ming you I tested with 100% vanilla morrowind, there might be a mod that is causing this, you might want to do a vanilla test or list out the mods you are using.

**Comment by Cloudy100** [comment]
Yeah, you were right. It seems the settings.cfg was messed up. My bad.

**Comment by XC3N** [comment]
Same thing happened to me after I upgraded the 0.49 dev build once.

Glad I could help!

**Comment by MontanaGandalf** [comment]
I have a similar problem. Can you tell me what was messed up in the settings.cfg?

**Comment by zesterer** [comment]
Ah, that's good to know. Which settings specifically caused this issue?
